app/domain/logs.rb

```diff
@@ -277,65 +277,65 @@ module AuthnJwt
 
       IssuerResourceNameConfiguration = ::Util::TrackableLogMessageClass.new(
         msg: "\"issuer\" value will be taken from '{0-resource-id}'",
-        code: "CONJ00054D"
+        code: "CONJ00054I"
       )
 
       RetrievedIssuerValue = ::Util::TrackableLogMessageClass.new(
         msg: "Retrieved \"issuer\" with value '{0}'",
-        code: "CONJ00055D"
+        code: "CONJ00055I"
       )
 
       ParsingIssuerFromUri = ::Util::TrackableLogMessageClass.new(
         msg: "Parsing \"issuer\" value from '{0}'...",
         code: "CONJ00056D"
       )
 
-      JWTAuthenticatorEntryPoint = ::Util::TrackableLogMessageClass.new(
-        msg: "Started JWT authentication flow for vendor '{0}'",
-        code: "CONJ00057D"
+      JwtAuthenticatorEntryPoint = ::Util::TrackableLogMessageClass.new(
+        msg: "Started JWT authentication flow for authenticator '{0-authenticator-name}'",
```

Comment:
can you provide an example? Is {0-authenticato-name} always going to be our (conjur) authenticators?
If so, then
```suggestion
        msg: "Started JWT authentication flow for {0-authenticator-name} Authenticator",
```

Reply:
@shulifink why not?
msg: "Started JWT authentication flow for '{0-authenticator-name}' Authenticator"

Reply:
@nessiLahav change to:
"Started authentication flow for authenticator 'authn-jwt'"

Reply:
fixed